skopt/learning/gaussian_process/gpr.py

```diff
@@ -186,7 +186,7 @@ def fit(self, X, y):
         if self.kernel is None:
             self.kernel = ConstantKernel(1.0, constant_value_bounds="fixed") \
                           * RBF(1.0, length_scale_bounds="fixed")
-        elif self.noise == "gaussian":
+        if self.noise == "gaussian":
             self.kernel = self.kernel + WhiteKernel()
```

Comment:
is there a way to check if `self.kernel` already contains a `WhiteKernel`? Or does someone know what happens when you add two `WhiteKernel`s together?

Reply:
Not easily anyway. However not much will happen - it will just make optimisation a bit slower as there is one extra useless hyperparameter to perform gradient descent over, but you will still get the same minimum